doc/sphinx/source/community/introduction.rst

```diff
@@ -294,14 +294,20 @@ What should be documented
 ~~~~~~~~~~~~~~~~~~~~~~~~~
 
 Any code documentation that is visible on
-`readthedocs <https://docs.esmvaltool.org>`__ should be well
-written and adhere to the standards for documentation for the respective
-language. Recipes should have a page in the *Recipes* section on
-readthedocs. This is also the place to document recipe options for the
-diagnostic scripts used in those recipes. Note that there is no need to
-write extensive documentation for functions that are not visible on
-readthedocs. However, adding a one line docstring describing what a
-function does is always a good idea.
+`docs.esmvaltool.org <https://docs.esmvaltool.org>`__
+should be well written and adhere to the standards for documentation for the
+respective language.
+Recipes should have a page in the :ref:`recipes` section.
+This is also the place to document recipe options for the diagnostic scripts
+used in those recipes.
+When adding a new recipe, please start from the
+`template <https://github.com/ESMValGroup/ESMValTool/blob/master/doc/sphinx/source/recipes/recipe_template.rst.template>`_
+and do not forget to add your recipe to the
+`<index <https://github.com/ESMValGroup/ESMValTool/blob/master/doc/sphinx/source/recipes/index.rst>`_.
+Note that there is no need to write extensive documentation for functions
+that are not visible on readthedocs.
+However, adding a one line docstring describing what a function does is
+always a good idea.
```

Comment:
I think we can ask for a bit more here.

```suggestion
Although it is not required to have extensive documentation for functions that are not visible on readthedocs, they should still have a descriptive docstring that explains what it does.
```

Reply:
Thanks for reviewing! The reason for asking for a one-line docstring is that I prefer not to have the full numpy style docstrings everywhere in the code, because the format is so lengthy that you can easily end up with much more documentation than code and that only makes the code less readable instead of more readable. For more complicated functions it probably makes sense to ask for more, but for functions that only have 5 lines of code it's pretty awkward reading if there are 25 lines of docstring above it. Do you have an idea on how to write that down nicely?

Reply:
Totally agree, but a one-line docstring for a function called `process_data` that says `"""Processes the data."""` is not helpful either. Above was already my attempt at writing this down nicely.

How about this:

```
Although it is not required to have extensive documentation for functions 
that are not visible on readthedocs, a short description in the docstring helps other 
contributors to understand what a function is intended to do and and what its capabilities are.
```

Reply:
Something like: "We seek a balance between being clear and being succinct. For short functions, a one-line docstring is usually okay, but more complex functions might require slightly more extensive documentation." ?

Reply:
Thanks! I tried to combine both suggestions.